Makefile

```diff
@@ -96,8 +98,7 @@ ifeq ($(EXCEPTIONS),y)
   # objects, so the linker ignores these objects entirely.  If you start
   # getting "multiple definition" and "undefined reference" errors,
   # there's probably a new ABI symbol we need to define ourselves.
-  CXXRUNTIME = $(shell $(CC) -print-file-name=libgcc_eh.a) \
-	  $(shell $(CC) -print-file-name=libsupc++.a)
+  CXXRUNTIME = $(CURDIR)/libgcc/libgcc_eh.a $(CURDIR)/libgcc/libsupc++.a
```

Comment:
I think this change could break with new GCC versions? My understanding is that those files are specific to the exact compiler version in use, so they need to be detected/extracted from the compiler at build time

Reply:
Ah okay, I don't think my installation of `x86_64-elf-gcc` came with `libgcc_eh.a` and `libsupc++.a` which is why I copied those files over from my athena box. I can revert these two lines and mention the workaround in the comments.

Reply:
Are you sure it doesn't come with them? We don't have to handle this right now, but I'd expect that they actually do get installed somewhere

Reply:
I don't think so, since when I remove the files I copied over, then the command `x86_64-elf-gcc -print-file-name=libgcc_eh.a` does not return anything.